src/main.js

```diff
@@ -209,6 +209,14 @@ table+table[aria-hidden=true] {
                 set: this._setValue.bind(this, valueDescriptor.set, updateValue)
             });
 
+            var valueAsDateDescriptor = Object.getOwnPropertyDescriptor(HTMLInputElement.prototype, "valueAsDate");
```

Comment:
On IE10-11 `Object.getOwnPropertyDescriptor(HTMLInputElement.prototype, "valueAsDate")` returns a null value, so `valueAsDateDescriptor.set` will fire an error. To fix that you should not use native browser property descriptor for `valueAsDate` but always provide implementation for getter and setter instead.